src/trainer/DataView.js

```diff
@@ -1,45 +1,35 @@
 import React from 'react';
-import Grid from '@material-ui/core/Grid';
+import { List, Card } from 'antd';
 
-const DataView = ({ dataName, imageInfos, offset, parentStyles }) => {
+const DataView = ({ dataName, imageInfos, offset }) => {
   // TODO(luke): Labels with the images
   const baseURL = 'https://storage.googleapis.com/elvos-public/processed';
```

Comment:
Thanks, good catch. IMO this should be a separate different PR, since this one is already getting large and it will take a lot of work to get this done and tested (we don't have two datasets in blueno-ml-files yet).

I've updated the name of the PR and created an issue for this #27